ninjareact/Dockerfile

```diff
@@ -0,0 +1,47 @@
+# Use an official Python runtime as a parent image
+FROM python:3.11-slim
+
+# Set environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+ENV DJANGO_ENV=production
```

Comment:
`DJANGO_ENV` should be managed at runtime. The other settings should not be used in production.